Makefile

```diff
@@ -119,6 +119,7 @@ test-go: libflux-go
 
 test-rust:
 	cd libflux && $(CARGO) test $(CARGO_ARGS) && \
+	$(CARGO) check # this checks for missing docs
 	$(CARGO) test --doc && \
 	$(CARGO) clippy $(CARGO_ARGS) -- -Dclippy::all
```

Comment:
How do `cargo check`,  `cargo test --doc`, and `cargo clippy` differ?

Reply:
`clippy` checks for some common errors and unidiomatic code. `test --doc` tests the examples within the docs. because we have `deny(missing_docs)` set, `check` will emit warnings for missing docs. (`cargo docs` would too, but we don't actually need to build the docs.)